controllers/storageclaim_controller.go

```diff
@@ -258,7 +264,7 @@ func (r *StorageClaimReconciler) reconcilePhases() (reconcile.Result, error) {
 				r.storageClaim.Status.Phase = v1alpha1.StorageClaimFailed
 				return reconcile.Result{}, nil
 			}
-		} else if err != nil && !errors.IsNotFound(err) {
+		} else if !errors.IsNotFound(err) {
```

Comment:
can you please move this change to a new commit?

Reply:
I observed it's taking more time to separate the commits rather than actually working on the commit or raising a PR. Raised a new PR along with some backlog items.

Reply:
already merged. can be resolved.